app/src/androidTest/java/de/test/antennapod/service/playback/PlaybackServiceTaskManagerTest.java

```diff
@@ -123,6 +129,42 @@ public void update(EventDistributor eventDistributor, Integer arg) {
         pstm.shutdown();
     }
 
+    @Test
+    public void testQueueUpdatedUponDownloadComplete() throws Exception {
+        final Context c = InstrumentationRegistry.getInstrumentation().getTargetContext();
+        { // Setup test data
+            List<FeedItem> queue = writeTestQueue("a");
+            FeedItem item = DBReader.getFeedItem(queue.get(0).getId());
+            FeedMedia media = new FeedMedia(item, "http://abc.test/acme.mp3", 12345, "audio/mp3");
```

Comment:
We are using example.com in most other tests. I would prefer to use that one instead because we know for sure that it is a placeholder url and will not become meaningful in the future.

Reply:
will do.